DOC: Modify index, overview and gettingstarted pages (#226)
equinor/fmu-settings: docs/src/index.md

> # FMU Settings documentation

Reviewer: This header should not be removed

> One of the main benefits of FMU Settings is to replace the global config .yaml files with a **GUI** (GUI = Graphical User Interface). In the FMU Settings GUI you will be able to :

Reviewer: a bit strange to me that we replace a file with a GUI. Suggestion:
```suggestion
One of the main benefits of FMU Settings is that it lets you manage global configuration through a graphical user interface (GUI) instead of editing YAML files manually. In the FMU Settings GUI, you can:
```

- set up FMU project metadata
- link FMU models to SMDA and other official metadata sources, including 
    - mapping of RMS stratigraphy to the SMDA stratigraphic column
    - *mapping of wells between RMS, Eclipse and SMDA (future feature)*
- keeping track of version history for your FMU project settings

If you are new to FMU Settings, a good place to start is:
1. Read the [Overview](overview.md) page to learn what FMU Settings is and how it is used.
2. Follow instructions on the [Getting started](getting_started.md) page to get started with FMU Settings for your FMU model(s).
3. Use the [GUI user guide](gui_user_guide.md) to configure and add metadata to your FMU model(s).

> 4. See also [Terminal commands](terminal_commands.md) for the most common terminal commands relevant for FMU Setting.

Reviewer: ```suggestion
4. See also [Terminal commands](terminal_commands.md) for the most common terminal commands relevant for FMU Settings.
```

Reviewer: this commit is gone because you do commit amend on the first commit and push force after that

# FMU Settings documentation

**FMU Settings** is a tool that will simplify the handling of FMU model metadata, meaning metadata related to the FMU model itself and metadata needed when you are exporting data objects from FMU models.

> One of the main benefits of FMU Settings is that it lets you manage global configuration through a graphical user interface (GUI) instead of editing YAML files manually. In the FMU Settings GUI, you can:

Reviewer 2: This felt a bit long and I think users may be confused by the term global configuration, they may think they can edit the entire global config file inside the application (future dream though 😄)
Maybe shorten it to
```suggestion
FMU Settings lets you manage static configuration through a graphical user interface (GUI) instead of editing files manually. In the FMU Settings GUI, you can:
```

Reviewer 3: Here is summary from Matthews slides on what FMU Settings is that might be usefull in this section:

"FMU Settings is a localhost web application that provides a user-friendly solution for static configuration in FMU projects. FMU Settings offers a unified way to connect FMU projects to other data sources within the company, i.e. SMDA."

- set up FMU project metadata
- link FMU models to SMDA and other official metadata sources, including 
    - mapping of RMS stratigraphy to the SMDA stratigraphic column

>     - *mapping of wells between RMS, Eclipse and SMDA (future feature)*

Reviewer 3: We should be even more clear here that this feature is not implemented yet to not confuse the user. Maybe something like `this feature is not implemented yet` instead of "future feature".

- keeping track of version history for your FMU project settings

> 

Reviewer 3: Would it be an idea to add a table or similar that shows how this FMU model metadata handling is done today and how it will be done with `fmu-settings`? I think it is important to be very clear about how fmu-settings will make things easier for the user so that they see the need for this right away. We need to sell this concept to the user from the start, so it's important that they see the need right way. 

When adding/selling a new tool there is often this perspective of a "Business case": What is the business need for this tool? Here it is common to focus on `pain points` in the current situation and how this new tool will address these pain points and make things easier for the user in the future. 

Maybe we could do something like
 
| Task | How it is done today | How it will be done in fmu-settings |
|------|---------------------|------------------------------------|
| Edit global config | Editing directly in yaml file | Editing through a form in a user interface  with validation |
| Add masterdata | Log in to SMDA, find field, copy paste to yaml (...) |  Integrate directly to SMDA from the GUI |
| Map stratigraphy | add text |  add text |

Reviewer 3: @krhart Have started on a business case here https://github.com/equinor/fmu-settings/issues/201
We can iterate on this and add that to the PR here when we have agreed on something.

If you are new to FMU Settings, a good place to start is:
1. Read the [Overview](overview.md) page to learn what FMU Settings is and how it is used.
2. Follow instructions on the [Getting started](getting_started.md) page to get started with FMU Settings for your FMU model(s).
3. Use the [GUI user guide](gui_user_guide.md) to configure and add metadata to your FMU model(s).
4. See also [Terminal commands](terminal_commands.md) for the most common terminal commands relevant for FMU Settings.

> If you find bugs, need help or want to get in touch, use:

Reviewer 3: Maybe also add `or have suggestion for new features` here for why the should contact us?

- <a href="https://equinor.enterprise.slack.com/archives/C09MFKN4NC9" target="_blank" rel="noopener noreferrer">#fmu-settings slack channel</a>
- <a href="https://fmu.equinor.com" target="_blank" rel="noopener noreferrer">FMU portal</a>

> FMU Settings is still being developed and some of the coming features will be:

Reviewer 3: We should maybe add a title to this coming features section. Something like `Upcoming features`
